appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/processor/BaseProcessor.java

```diff
@@ -76,6 +76,8 @@ public OpenAPI process(OpenAPI api, OpenApiConfiguration config) {
 
         // Set the OpenAPI version if it hasn't been set
         if (api.getOpenapi() == null) {
+            // FIXME: change to 3.1.0 when MP TCK is upgraded
+            // api.setOpenapi("3.1.0");
             api.setOpenapi("3.0.0");
```

Comment:
Do we need to wait the upgrade? I saw that you changed version on other parts

Reply:
3.1.0 breaks the MP TCK.